src/style.css

```
@@ -1,3 +1,74 @@
+:root {
+  --font: Roboto, sans-serif;
+  --main-color: #000;
+  --color-links: rgb(0, 172, 220);
+  --size: 12px;
+  --bg-color: #fff;
+  --weight: 500;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  position: relative;
+  font-family: var(--font);
+  line-height: 60px;
+  color: var(--main-color);
+  font-size: var(--size);
+  font-weight: var(--weight);
```

```suggestion
  line-height: 60px;
  color: var(--main-color);
```
u can remove these properties if u have set the default properties for the main fonts(in all the cases)